src/popup/views/app.vue

```diff
@@ -185,9 +185,10 @@ export default {
       ].filter(Boolean).join(' ');
     },
     failureReasonText() {
+      const { injectable, blacklisted } = store;
       return (
-        !store.injectable && i18n('failureReasonNoninjectable')
-        || store.blacklisted && i18n('failureReasonBlacklisted')
+        !injectable && i18n('failureReasonNoninjectable')
+        || blacklisted && `${i18n('failureReasonBlacklisted')} <code>${blacklisted}</code>`
```

Comment:
`blacklisted` needs to be escaped when using `v-html`.

Reply:
Got it. Switched to a more declarative method.